locale/de/base.cfg

```diff
@@ -1,18 +1,18 @@
 [mod-name]
-LogisticTrainNetwork=LTN - Logistisches Bahn Netzwerk
+LogisticTrainNetwork=LTN – Logistisches Zugnetzwerk
 
 [mod-description]
-LogisticTrainNetwork=Fügt neue Haltestellen hinzu die ein frei konfigurierbares Logistik Netzwerk aufspannen.
+LogisticTrainNetwork=Fügt neue Haltestellen hinzu, die ein hochkonfigurierbares und vollautomatisiertes Logistiknetzwerk bilden.
 
 [entity-name]
-logistic-train-stop=Logistik Haltestelle
-logistic-train-stop-input=Logistik Haltestelle Signaleingang
-logistic-train-stop-output=Logistik Haltestelle Signalausgang
-ltn-port=Logistik Hafen
+logistic-train-stop=Logistische Zughaltestelle

```

Comment:
Keine Ahnung wie es in Deutschland ist, aber In Österreich ist es einfach nur Haltestelle.

Reply:
Wie genau jetzt? Soll es jetzt nur "Haltestelle" oder "Zughaltestelle" heißen?

Reply:
Haltestelle